ui/apos/components/AposExportModal.vue

```diff
@@ -242,9 +244,11 @@ export default {
         }
       });
       this.checkedRelatedTypes = this.relatedTypes;
-      const height = this.checkedRelatedTypes.length
+      const height = this.relatedTypes.length
         ? this.checkedRelatedTypes.length * CONTAINER_ITEM_HEIGHT + CONTAINER_DESCRIPTION_HEIGHT
         : CONTAINER_MINIMUM_HEIGHT;
+
+      await this.$nextTick();
```

Comment:
It worked until now because we make an http request before. But without the ref container isn't accessible.
Just a guard to make sure it'll work whatever time takes the  request.